app/routes.py

```diff
@@ -1,2 +1,291 @@
-from flask import Blueprint
+from flask.json import jsonify
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+from datetime import date, datetime
```

Comment:
It doesn't look like `date` is used anywhere, so we can remove that.

```suggestion
from datetime import datetime
```